app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -53,6 +63,32 @@ class WireActivityViewModel @Inject constructor(
         }
     }
 
+    init {
+        viewModelScope.launch {
+            withContext(dispatchers.io()) {
+                val currentSessionResult = currentSessionUseCase()
+                if (currentSessionResult is CurrentSessionResult.Success) {
+                    listenForNotifications(currentSessionResult.authSession.userId)
```

Comment:
Also if the user is logged out when running the app for the first time, this will return `Failure` and on successful login it doesn't seem that notifications listener gets enabled either.